src/tests/integration/steps/posting-feedback.ts

```diff
@@ -48,31 +52,30 @@ export function postingFeedback(mocks: Mocks, opts: { handle: string; result: Po
     })
 
     if (opts.result === 'success') {
-      it('launches a new tab with the tweet upon completion and clears the existing feedback', async () => {
-        await mocks.whenState(state => !ensureActiveTab(state).feedbackState.isTweeting)
-        expect(mocks.chrome.tabs.create).to.have.been.calledOnceWithExactly({
-          url: 'https://t.co/sometweethash',
-          active: true,
-        })
-
-        const spans = mocks.app().querySelectorAll('.twitter-interface > .DraftEditor-root span[data-text="true"]')
-
-        expect(spans).to.have.length(2)
-        expect(spans[0]).to.have.property('innerHTML', opts.handle)
-        expect(spans[1]).to.have.property('innerHTML', ' ')
+      it('creates a new tab with the tweet upon completion and clears the existing feedback', async () => {
+        const state = await mocks.whenState(state => !ensureActiveTab(state).feedbackState.isTweeting)
+        expect(mocks.chrome.tabs.create).to.have.callCount(1 + priorChromeTabsCreateCallCount)
+        expect(mocks.chrome.tabs.create.lastCall.args).to.eql([
+          {
+            url: 'https://t.co/sometweethash',
+            active: true,
+          },
+        ])
+        const activeTab = ensureActiveTab(state)
```

Comment:
Wouldn't this return the new tab with the tweet? We should also check the `editorState` of the previous tab

Reply:
We test that the feedback is posted with the correct status in the assertions above. Here we assert that the feedback is cleared. Is there something else, you're calling for?

Reply:
I thought we are "opening" a new tab while testing. If that was the case, this `ensureActiveTab` call would return the new tab. Checking the status of the new tab is nice to have, but we need to check that the old status was erased. Since we stay in the same tab while testing, my comment doesn't make much sense.